pages/news/2020-XX-XX-new-in-2-3-serato-support.html

```diff
@@ -0,0 +1,78 @@
+
+If you have USB drives with a [portable Serato library](https://support.serato.com/hc/en-us/articles/202304844-Using-a-USB-external-hard-drive-for-your-portable-library) on it, we've got you covered as well:
+Libraries and crates on external USB drives will work on all supported platforms - even on Linux!
+
+{# TODO: Insert screenshot of Serato library feature once the skins are finalized. #}
+
+Sorting tracks in your library is importing, but preparing [cue points](https://support.serato.com/hc/en-us/articles/226518228-Cue-Points) in tracks is probably the most time-consuming task.
+Serato saves them in special file tags along with the other track metadata like title, artist and so on.
+It took a months of work, but we finally manage to [reverse-engineer the binary formats](https://github.com/Holzhaus/serato-tags) that are used by Serato to a degree that allows us to parse this information and use it in Mixxx.
+
+## Importing your Hotcues
+
+This means you can import the positions, labels and colors of your hotcues into Mixxx automatically when you load the track for the first time.
+
+{# TODO: Insert side-by-side screenshot of hotcues in Mixxx and Serato once the skins are finalized. #}
+
+If you already added a track to your Mixxx library before support for reading Serato's hotcues was added, you can trigger a reimport of the metadata via the track context menu.
+Note that this will clear your existing cuepoints in Mixxx if the track has any Serato hotcues.
+
+{# TODO: Add screenshot of reimport metadata context menu entry. #}
+
+Unfortunately, a myriad of encoders and decoders for lossy formats such as MP3 and M4A/AAC exist, which leads to a situation where all decodes detect slightly different track start and end times for files from different sources.
+Hence, your cues might end up shifted by a few milliseconds.
+
+We've done our best to mitigate the problem, but if you experience issues we'd appreciate if you get in touch with us on Zulip and work with us to make the offset correction more accurate.
+Additionally, we've added a way to shift all cues for a track at once. This makes it possible to fix the cue positions if Mixxx fails to determine the correct offsets.
+
+## Support for other Serato metadata
+
+Mixxx 2.3 will also import saved loops from Serato, but proper support for saving and restoring loops is scheduled for 2.4.
+For the time being, we just import these loops into the Mixxx database and allows using their start position as regular hotcues.
+That way, you can start using your loops once 2.4 is out without the need to reimport your metadata (which would undo your cue modifications in Mixxx).
+
+Other Serato metadata that we parse include the track color and the BPM lock status.
+Serato also stores other information like the overview waveform image and the beatgrid in the file tags, but we don't import that information in Mixxx 2.3.
```

Comment:
...import these information... I think.

Reply:
I think the original sentence is correct?